metomi/rose/host_select.py

```diff
@@ -393,23 +405,20 @@ def select(
             # build host-select-client command
             command = []
             if not self.is_local_host(host_name):
-                command_args = []
-                command_args.append(host_name)
-                command = self.popen.get_cmd("ssh", *command_args)
+                command = self.popen.get_cmd("ssh", host_name)
 
-            # pass through CYLC_VERSION to support use of cylc wrapperf script
+            # pass through CYLC_VERSION to support use of cylc wrapper script
             try:
                 import cylc.flow
             except ModuleNotFoundError:
                 pass
             else:
                 command.extend([
                     'env',
-                    f'CYLC_VERSION={cylc.flow.__version__}'
+                    f'CYLC_VERSION={cylc.flow.__version__}',
+                    f'CYLC_ENV_NAME=cylc-{cylc.flow.__version__}'
```

Comment:
This should take the value of `$CYLC_ENV_NAME` if it is defined, otherwise it should be omitted.

Reply:
Done: https://github.com/metomi/rose/compare/43220b91f648379dc34361160edabf5e816ef348..e8bd96f537262d2748f17fe15096fe2126c4bd22